test/unit/h5.cpp

```diff
@@ -239,6 +240,59 @@ TEST(h5dump, ublas_matrix_real_part_rect) { // NOLINT
   }
 }
 
+template <typename T1, typename T2, int N, int M, int K, int L>
```

Comment:
Tole se ponavlja. Naj bodo vse te funkcije na enem mestu, recimo v compare.hpp, ki si ga ze zacel pisati. Kaksno podobno funkcijo za priperjave sem tudi jaz morda ze kje napisal in se tudi lahko premakne tja.

Reply:
zato sem compare.hpp naredil ja, ampak mi ni delalo v redu, nekaj z linkanjem se mi zdi, moram še probat enkrat

Reply:
Morda manjka #pragma one ali header include guards? Ali pa kaksna navadna funkcija ni oznacena z inline?

Reply:
problem je bil, da je ublas definiran v NRG namespaceu in nisem pomislil na to

Reply:
oz. da je bil using namespace boost::numeric v NRG